CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## v2.13.2dev
 
+- Handle more complete list of possible git URL forms (ssh:// and ftp:// prefixes specifically)
+
```

Comment:
Could we remove this? I used the bot to add the line with the same format we usually use, feel free to update the line which the bot added :)